circle.yml

```diff
@@ -57,14 +57,14 @@ test:
     - touch $CIRCLE_TEST_REPORTS/xunit/server-tests.xml
   override:
     # Run client tests
-    - ./node_modules/.bin/mocha --timeout 30000 -R xunit dist/test.js > $CIRCLE_TEST_REPORTS/xunit/client-tests.xml:
```

Comment:
I bumped the timeouts up further. Because why be stingy... Also the server tests sometimes took slightly over 30 seconds.